build.gradle

```diff
@@ -28,28 +28,29 @@ repositories {
 }
 
 dependencies {
-    compileOnly("org.embulk:embulk-api:0.10.5") {
+    compileOnly("org.embulk:embulk-api:0.10.12") {
         exclude group: "*"
     }
-    compileOnly("org.embulk:embulk-core:0.10.5") {
+    compileOnly("org.embulk:embulk-api:0.10.12") {
```

Comment:
i think it is redundant.

Reply:
Ah, thanks for the good catch! I intended this to be `embulk-Spi`, but I mistook... Fixed in: 204fdd95870e05152f91ce896894654a291fcff1